### PR TITLE
feat: [ANDROAPP-3516] New navigation in tei dashboard

### PR DESCRIPTION
**Pull request by Balcan**
## Description
[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-3516)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by ferdyrod — APPROVED** [review]

File: app/src/main/java/org/dhis2/utils/customviews/DhisBottomNavigationBar.kt [hunk 1]
```diff
@@ -34,6 +36,7 @@ class DhisBottomNavigationBar @JvmOverloads constructor(
     private val currentItemIndicatorColor: Int
     private val itemIndicatorSize: Float
     private val itemIndicatorDrawable: Drawable?
+    private var currentItemId: Int = -1
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/utils/customviews/DhisBottomNavigationBar.kt:39]: isn't the first navigation_details and not -1?

File: app/src/main/java/org/dhis2/usescases/teiDashboard/adapters/DashboardPagerAdapter.kt [hunk 2]
```diff
@@ -39,9 +58,35 @@ class DashboardPagerAdapter(
...
+    }
+
+    fun getNavigationPagePosition(navigationId: Int): Int {
+        return when (navigationId) {
+            R.id.navigation_details -> if (isLandscape()) -1 else 0
+            R.id.navigation_events -> -1
+            R.id.navigation_analytics -> if (isLandscape()) 0 else 1
+            R.id.navigation_relationships -> if (isLandscape()) 1 else 2
+            R.id.navigation_notes -> if (isLandscape()) 2 else 3
+            else -> -1
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/adapters/DashboardPagerAdapter.kt:83]: Can you use constants? It's very hard to guess what 2 3 1 and 0 

Thankss !!!

File: app/src/main/java/org/dhis2/utils/customviews/DhisBottomNavigationBar.kt (hunk 1 above)
Comment (JaimeToca) [reply app/src/main/java/org/dhis2/utils/customviews/DhisBottomNavigationBar.kt:39]: ? @Balcan

File: app/src/main/java/org/dhis2/usescases/teiDashboard/adapters/DashboardPagerAdapter.kt [hunk 3]
```diff
@@ -20,6 +22,23 @@ class DashboardPagerAdapter(
...
+        } else {
+            createPortraitFragment(position)
+        }
+    }
+
+    private fun createLandscapeFragment(position: Int): Fragment {
+        return when (position) {
+            0 -> IndicatorsFragment()
+            1 -> RelationshipFragment()
+            2 -> NotesFragment.newTrackerInstance(currentProgram!!, teiUid)
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/adapters/DashboardPagerAdapter.kt:36]: Could you changes these too? 

Thanks

**Review by JaimeToca — APPROVED** [review]
@Balcan Everything okay, there are a few magic numbers you forgot to change. check it out pls

**Pull request merged**